.github/workflows/cypress.yaml

```
@@ -0,0 +1,86 @@
+      
+      - name: Run Cypress Tests with Dev Server
+        uses: cypress-io/github-action@v6
+        with:
+          start: npm run serve
+          wait-on: 'http://localhost:4321'
+          wait-on-timeout: 120
+          config-file: cypress.config.ts
+          install: false
+        env:
+          CYPRESS_BASE_URL: http://localhost:4321
+          REACT_APP_TALAWA_URL: http://localhost:4000/graphql
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)